test/pad_table.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html> <html>
+    <head lang="en"> <meta charset="UTF-8"> <title></title> </head>
+    <body> <h1>This is a test document</h1> With some text, <code>code</code>, <b>bolds</b> and <i>italics</i>.  <h2>This is second header</h2> <p style="display: none">Displaynone text</p> 
+    <table>
+        <tr> <th>Header 1</th> <th>Header 2</th> <th>Header 3</th> </tr>
+        <tr> <td>Content 1</td> <td>Content 2</td> <td><img src="http://lorempixel.com/200/200" alt="200"/> Image!</td> </tr>
+        <tr> <td>Content 1</td> <td>Content 2</td> <td><img src="http://lorempixel.com/200/200" alt="200"/> Image!</td> </tr>
+    </table>
+
+
+    <table> <tr> <th>Header 1</th> <th>Header 2</th> <th>Header 3</th> </tr>
+        <tr> <td>Content 1</td> <td>Content 2</td> <td><img src="http://lorempixel.com/200/200" alt="200"/> Image!</td> </tr>
+        <tr> <td>Content 1</td> <td>Content 2</td> <td><img src="http://lorempixel.com/200/200" alt="200"/> Image!</td> </tr>
+    </table>
+
+some content within the table<br>
```

Comment:
Did you mean "between tables"? Because this line is not actually within a table.